### PR TITLE
Update telling-git-about-your-signing-key.md

### PR DESCRIPTION
**Pull request by xty**
Corrected instructions for setting the `GPG_TTY` environment variables on GNU/Linux

### Why:

The current instructions for setting `GPG_TTY` is unsuited for GNU/Linux. Unlike on macOS where interactive shells are also login shells, shells lauched by terminal emulators on GNU/Linux are non-login shells by default. As a result, neither `.bash_profile` nor `.profile` will be executed when they are started normally. See [*Bash Reference Manual*](https://www.gnu.org/software/bash/manual/html_node/Bash-Startup-Files.html) for a fuller explanation.

### What's being changed:

The relevant instructions have been updated to align with the standard practice of putting initialization commands in `.bashrc` on GNU/Linux. Please note that [*the GnuPG Manual*](https://www.gnupg.org/documentation/manuals/gnupg/Invoking-GPG_002dAGENT.html#:~:text=lines%20to%20your-,.bashrc,-or%20whatever%20initialization) also recommends that:
> You should always add the following lines to your **`.bashrc`** or whatever initialization file is used for all shell invocations:
> ```shell
> GPG_TTY=$(tty)
> export GPG_TTY
> ```

### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@xty 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Review by Susanto2 — COMMENTED** [review]
B cearfull

File: content/authentication/managing-commit-signature-verification/telling-git-about-your-signing-key.md [hunk 1]
```diff
@@ -96,16 +96,10 @@ If you have multiple GPG keys, you need to tell Git which one to use.
 {% data reusables.gpg.list-keys-with-note %}
 {% data reusables.gpg.copy-gpg-key-id %}
 {% data reusables.gpg.paste-gpg-key-id %}
-1. To add your GPG key to your bash profile, run the following command:
-  ```shell
-  $ if [ -r ~/.bash_profile ]; then echo 'export GPG_TTY=$(tty)' >> ~/.bash_profile; \
-    else echo 'export GPG_TTY=$(tty)' >> ~/.profile; fi
+1. To add your GPG key to your `.bashrc` startup file, run the following command:
+  ```console
```
Comment (vgrl) [inline content/authentication/managing-commit-signature-verification/telling-git-about-your-signing-key.md:100]:
```suggestion
  ```bash
```

**Review by vgrl — DISMISSED** [review]
This is a great contribution, thank you @xty !

One minor change to use the `bash` code block type instead of `shell` (we don't support the `console` type). I'll push this change and then the PR will be ready to merge 🎉 .

**Review by vgrl — APPROVED** [review]
Looks good in preview.

**Review by ALTURKA — COMMENTED** [review]
Duplicate of #

File: content/authentication/managing-commit-signature-verification/telling-git-about-your-signing-key.md (hunk 1 above)
Comment (xty) [reply content/authentication/managing-commit-signature-verification/telling-git-about-your-signing-key.md]: FYI, `console` is just an alias for `ShellSession` :smile: 
https://github.com/github/linguist/blob/c34f887f48d81e2ed42b59825fa5fe12d3918d63/lib/linguist/languages.yml#L6392

**Pull request merged**